File: src/dataset/python/resistor-color_exercism-fruit-in/claude/ClaudeSonnet4_resistor_color_v4.py
_COLORS = [
    "black",
    "brown",
    "red",
    "orange",
    "yellow",
    "green",
    "blue",
    "violet",
    "grey",
    "white",
]

_COLOR_INDEX_MAP = {color: index for index, color in enumerate(_COLORS)}


def color_code(color):
    return _COLOR_INDEX_MAP[color]


def colors():
    return _COLORS

File: src/dataset/Java/resistor-color_exercism-uzilan/claude/ClaudeSonnet4_ResistorColor_v4.java
import java.util.HashMap;
import java.util.Map;

class ResistorColor {

    private static final String[] COLOR_NAMES = {
        "black", "brown", "red", "orange", "yellow", 
        "green", "blue", "violet", "grey", "white"
    };
    
    private static final Map<String, Integer> COLOR_CODE_MAP = new HashMap<>();
    
    static {
        for (int i = 0; i < COLOR_NAMES.length; i++) {
            COLOR_CODE_MAP.put(COLOR_NAMES[i], i);
        }
    }

    int colorCode(String color) {
        return COLOR_CODE_MAP.get(color);
    }

    String[] colors() {
        return COLOR_NAMES.clone();
    }
}